src/test/scala/uk/gov/nationalarchives/tdr/validation/JsonSchemaValidatorsSpec.scala

```diff
@@ -0,0 +1,47 @@
+package uk.gov.nationalarchives.tdr.validation
+
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import uk.gov.nationalarchives.tdr.validation.schema.JsonSchemaDefinition.BASE_SCHEMA
+import uk.gov.nationalarchives.tdr.validation.schema.JsonSchemaValidators
+
+class JsonSchemaValidatorsSpec extends AnyWordSpec with Matchers {
+
+  "JsonSchemaValidators" when {
+    "validating using base schema" should {
+      "produce array error for foi_exemption_code" in {
```

Comment:
What causes the error. Is it because its an unrecognized FOI exemption code. Should add something into the test title to make clear what is causing the error.